API/Tensorflow/src/Tensor.luna

```diff
@@ -58,11 +58,6 @@ class Tensor a:
         dims = self.dims
         flat = self.toFlatList
 
-        # Internal.
```

Comment:
Why is this documentation removed?

Reply:
Because comments inside functions break something in Luna and these weren't too useful anyway